mc2/controllers/base/models.py

```diff
@@ -54,6 +54,8 @@ class Controller(PolymorphicModel):
             'is automatically set each time the item is saved.')
     )
 
+    webhook_token = models.UUIDField(null=True)
```

Comment:
Can we pregenerate a UUID4? Do we want to do that in the UI or on the model level?